### PR TITLE
Move common stream properties section. Add task one

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes https://github.com/spring-cloud/spring-cloud-dataflow/issues/1319
Fixes https://github.com/spring-cloud/spring-cloud-dataflow/issues/1386

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 1]
```diff
@@ -174,6 +174,29 @@ be `TaskLauncher` implementation specific.
...
+
+```
+--spring.cloud.dataflow.applicationProperties.task.foo=bar
+--spring.cloud.dataflow.applicationProperties.task.fizz=bar2
+```
+
+This will cause the properties `foo=bar` and `fizz=bar2` to be passed to all the launched applications.
+
+[NOTE]
+Properties configured using this mechanism have lower precedence than task deployment properties.
```
Comment (cppwfs) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: 'launch' properties instead of 'deployment' properties.

Comment (ericbottard) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Hmmm, not sure. While I see what you mean, it is indeed deployment properties, as in [here](https://github.com/spring-cloud/spring-cloud-deployer/blob/master/spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/core/AppDeploymentRequest.java#L62)

Comment (cppwfs) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: My bad.  Please disregard.

**Comment by markpollack** [comment]
merged as f0f076289a0e5e6e15c10dc8b7d8f5e9f300b333

**Pull request closed without merging**